deps/wazuh_testing/wazuh_testing/tools/logging.py

```diff
@@ -132,3 +133,16 @@ def error(self, message):
     def critical(self, message):
         """Log CRITICAL message"""
         self.logger.critical(message)
+
+
+LOGGING_LEVELS = {
```

Comment:
Suggest:

- Default_level: ERROR
- `V`: INFO
- `VV`: DEBUG

Reply:
It should be complicated handling this structure due to `pytest_logger_config` and `--logger-logsdir` option. My suggestion is to use the following.

```
LOGGING_LEVELS = {
    'V': logging.CRITICAL,
    'VV': logging.ERROR,
    'VVV': logging.WARNING
}
```
Where V is the default value.